File: src/main/java/com/api/air_quality/model/CommentModel.java
package com.api.air_quality.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@ToString

@Document(collection = "comment")
public class CommentModel {
    @Id
    private String id;
    private String blogId;
    private String name;
    private String email;
    private String profile;
    private String date;
    private String comment;
    private Object reply;
    private Integer like;
}

File: client/src/app/shared/model/Comment.model.ts
export interface CommentModel {
  id: any;
  blogId: any;
  name: any;
  email: any;
  profile: any;
  date: any;
  comment: any;
  reply: {
    id: any;
    commentId: any;
    name: any;
    email: any;
    profile: any;
    date: any;
    replyComment: any;
  }[];
  like: any;
}
